OFEP-flagd-grpc-sync.md

```diff
@@ -0,0 +1,38 @@
+## OFEP: Add gRPC sync support to flagd
+
+## State: APPROVED
```

Comment:
Is this already approved?

Reply:
I had added a comment to make sure we set the state before merging but this was updated prematurely.

Reply:
No, not yet. I changed this after few approvals and I am still waiting for more reviews.